reader-osm/src/test/java/com/graphhopper/reader/osm/GraphHopperOSMTest.java

```diff
@@ -573,7 +573,7 @@ public void testFootOnly()
         assertEquals(2, instance.getGraphHopperStorage().getAllEdges().getMaxId());
 
         // A to E only for foot
-        GHResponse grsp = instance.route(new GHRequest(11.1, 50, 11.2, 52.01).setVehicle("foot"));
+        GHResponse grsp = instance.route(new GHRequest(11.1, 50, 11.199999, 52).setVehicle("foot"));
```

Comment:
Actually I don't really understand that fix. I looked inside the corresponding xml file, but could not find the points?